HACKERS

```diff
@@ -235,18 +235,12 @@ perl versions).
 C<ivers()> is used in the C<=tests> section to overcome this. and constructs
```

Comment:
I think the `Version numbers` should be in the D-P doc, not hidden in the HACKERS file

Reply:
`{ VERSION }` is for HACKERS /internal
whereas `PERL_VERSION_` is a public API / methods for people using ppport.h